spec.bs

```diff
@@ -138,6 +138,14 @@ dictionary PADebugModeOptions {
 };
 </xmp>
 
+Issue: Per the <a href=https://www.w3.org/TR/design-principles/#numeric-types>
+  Web Platform Design Principles</a>, we should consider switching `long` to
+  `[EnforceRange] long long`.
+
+Issue: {{PrivateAggregation/enableDebugMode(options)}}'s argument should not
```

Comment:
Note that fixing this to match the implementation (i.e. an optional argument with no default value) causes validation issues in the spec. So, we might need to change it by making debugKey optional instead.